src/mscorlib/shared/System/Memory.cs

```diff
@@ -6,8 +6,7 @@
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-using EditorBrowsableAttribute = System.ComponentModel.EditorBrowsableAttribute;
-using EditorBrowsableState = System.ComponentModel.EditorBrowsableState;
+using System.ComponentModel;
```

Comment:
I preferred it the original way. `System.ComponentModel` is not an obvious namespace to import into a file like `Memory.cs` - spelling it out the types makes it clear why we're importing it.